### PR TITLE
Fixed Links on #3160

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving issues on #3160

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/5-envision-plan-my-users-experience.md [hunk 1]
```diff
@@ -160,9 +160,9 @@ device strategy is optimal.
 > [!NOTE]
 > At this time, we recommend using audio devices that were certified
 through the Skype for Business Certification program. To find devices certified
-under this program, see the [USB Devices Certified for Skype for
-Business](http://partnersolutions.skypeforbusiness.com/solutionscatalog/personal-peripherals-pcs)
-solutions catalog.
+under this program, see the [Teams certified devices](https://products.office.com/microsoft-teams/across-devices/devices) and [USB Skype For Business Devices](https://docs.microsoft.com/SkypeForBusiness/certification/devices-usb-devices).
```
Comment (JohanFreelancer9) [inline Teams/5-envision-plan-my-users-experience.md]:
```suggestion
under this program, see the [Teams certified devices](https://products.office.com/en-us/microsoft-teams/across-devices/devices) and [USB Skype For Business Devices](https://docs.microsoft.com/SkypeForBusiness/certification/devices-usb-devices).
```

**Comment by scanum** [comment]
@rowille PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**